recipe/meta.yaml

```diff
@@ -27,7 +25,7 @@ requirements:
   run:
```

Comment:
I think `wheel` is missing in `host`

Reply:
I believe the way the maintainers have currently written the `setup_sklearnex.py` file causes problems when trying to run `pip install ....` instead of the old `setup.py install` approach to the build command. As a result, I don't think `pip` and `wheel` are necessary in this recipe yet.

Reply:
You are right.